cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -2737,6 +2737,58 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRunModeUpgradeScenarios(c *C)
 	}
 }
 
+func (s *initramfsMountsSuite) TestInitramfsMountsRunModeUpdateBootloaderVars(c *C) {
```

Comment:
can we also have a test that shows we clear junk/failed update variable values too?